package.json

```
@@ -1,29 +1,39 @@
 {
-  "name": "trie-search",
-  "author": "Joshua Jung <joshua.p.jung@gmail.com>",
+  "name": "@commitd/trie-search",
```

our npm package is `@committed`

```suggestion
  "name": "@committed//trie-search",
```